tests/tabular/checks/model_evaluation/single_dataset_performance_test.py

```diff
@@ -157,6 +157,18 @@ def test_regression(diabetes_split_dataset_and_model):
         assert_that(metric_col['Value'].iloc[0], instance_of(float))
 
 
```

Comment:
Please add a test to catch the warning and maybe also a check with the positive score to make sure it doesn't mess anything.